xonsh/ptk_shell/updator.py

```diff
@@ -157,37 +157,53 @@ def submit_section(
 class PromptUpdator:
     """Handle updating multiple AsyncPrompt instances prompt/rprompt/bottom_toolbar"""
 
-    def __init__(self, session: PromptSession):
+    def __init__(self, shell):
+        from xonsh.ptk_shell.shell import PromptToolkitShell
```

Comment:
Does it really change the uptime if we import this normally? If not pls move this to the top of the file

Reply:
this is to prevent circular import